pinot-core/src/main/java/org/apache/pinot/core/api/ServiceAutoDiscoveryFeature.java

```diff
@@ -68,8 +69,8 @@
  * pinot-integration-tests/src/main/java/org/apache/pinot/broker/integration/tests/BrokerTestAutoLoadedService.java
  * </code>
  */
-public class BrokerServiceAutoDiscoveryFeature implements Feature {
-    private static final Logger LOGGER = LoggerFactory.getLogger(BrokerServiceAutoDiscoveryFeature.class);
+public class ServiceAutoDiscoveryFeature implements Feature {
```

Comment:
What will happen if we have PinotServiceManager deploying controller/broker/server in the same JVM and all of them trying to load this `ServiceAutoDiscoveryFeature`?